### ISSUE TITLE
iOS height not reducing in modal.

### ISSUE THREAD

**Issue by ChronixPsyc** [jackmoore/autosize#284]
Hi,

I'm using Mobile Angular UI for a project I'm working on and needed to resize a textarea inside a modal when the text grows larger than the minimum 250px height.

The textarea grows fine, but when reducing the amount of line breaks it does not reduce. I have used the autosize:update event and also autosize:resize event to console log to make sure that they are both being called and they are, the issue is that the height is not being reduced in them.

**Comment by jackmoore** [comment]
Does your use of Mobile Angular UI matter?  I'm probably not going to try to reproduce this issue, but if you setup a demo with only the parts necessary to reproduce the issue I would look into it.

**Comment by ChronixPsyc** [comment]
@jackmoore The only reason I mentioned Mobile Angular UI was we have used other plugins in the past that were not working nicely with Mobile Angular UI.

But I've actually figured out the issue to this and it was the fact that my textarea still had transitions on the height.... Sorry! I must have originally misread the line of text that said transistions **won't** work.

It's now working as intended and is a really handy piece of code
